### PR TITLE
Added support for creating database with shared throughput for Azure Cosmos DB

### PR DESCRIPTION
**Pull request by srinathnarayanan**

### CODE REVIEW

File: src/command_modules/azure-cli-cosmosdb/azure/cli/command_modules/cosmosdb/custom.py [hunk 1]
```diff
@@ -256,9 +256,9 @@ def cli_cosmosdb_database_list(client):
     return list(client.ReadDatabases())
 
 
-def cli_cosmosdb_database_create(client, database_id):
+def cli_cosmosdb_database_create(client, database_id, throughput=None):
     """Creates an Azure Cosmos DB database """
-    return client.CreateDatabase({'id': database_id})
+    return client.CreateDatabase({'id': database_id}, {'offerThroughput': throughput})
```
[the remark this answers is not in the document]
Comment (srinathnarayanan) [reply src/command_modules/azure-cli-cosmosdb/azure/cli/command_modules/cosmosdb/custom.py]: The 'offerThroughput' parameter is defined for the Python SDK. It is checked for and a 'x-ms' header is added during processing the request, by the SDK.

[the remark this answers is not in the document]
Comment (kirankumarkolli) [reply src/command_modules/azure-cli-cosmosdb/azure/cli/command_modules/cosmosdb/custom.py]: Closed offline.

File: src/command_modules/azure-cli-cosmosdb/azure/cli/command_modules/cosmosdb/_params.py [hunk 2]
```diff
@@ -52,3 +52,6 @@ def load_arguments(self, _):
         c.argument('partition_key_path', help='Partition Key Path, e.g., \'/properties/name\'')
         c.argument('indexing_policy', type=shell_safe_json_parse, completer=FilesCompleter(), help='Indexing Policy, you can enter it as a string or as a file, e.g., --indexing-policy @policy-file.json)')
         c.argument('default_ttl', type=int, help='Default TTL')
+
+    with self.argument_context('cosmosdb database') as c:
+        c.argument('throughput', type=int, help='Offer Throughput')
```
Comment (tjprescott) [inline src/command_modules/azure-cli-cosmosdb/azure/cli/command_modules/cosmosdb/_params.py]: What are the units for this?  They should be called out in the help text.

Comment (srinathnarayanan) [reply src/command_modules/azure-cli-cosmosdb/azure/cli/command_modules/cosmosdb/_params.py]: added units in latest iteration

**Pull request merged**